python/lsst/ci/prepare.py

```diff
@@ -264,6 +281,28 @@ def fetch(self, product):
 
         """
 
+        for i in range(self.retries - 1):
```

Comment:
Does this not run by default? (`retires = 1`)

Reply:
It doesn't, the parameter is misnamed, it should be something like `tries`.